ts/observability-kit/package.json

```diff
@@ -0,0 +1,44 @@
+{
+  "name": "@hilla/observability-kit",
+  "version": "0.1.0",
```

Comment:
Let's call this package `@hilla/observability-kit-client` to use the same convention we have for SSO Kit. Also, the version should reflect the Java one i.e. `2.1-SNAPSHOT` now.

Reply:
What about split it to `@vaadin/observability-kit-klient` and `@hilla/observability-kit-client`? We could reuse `init` function along with `FrontendErrorInstrumentation` in the Flow implementation

Reply:
Btw, we cannot use `SNAPSHOT` in the version for npm. We should probably use `2.1-alpha.{x}` with some number instead of `{x}`. For now, I gonna use `2.1-alpha.0`

Reply:
Good idea, let's do this: the two clients sharing the `init` function. For the version, let's then use the same we're about to release, which is `2.1.0.beta3`

Reply:
Is it ok for Flow client use the `@hilla` package? Or it's better to create two packages (I would choose this variant).

Reply:
How much they differ at this point? The Flow one uses the [`ObservabilityHandler`](https://github.com/vaadin/observability-kit/blob/f44f8df5959300ca51426efb61fe0c85c0a53a32/observability-kit-starter/src/main/java/com/vaadin/observability/ObservabilityHandler.java#LL37C43-L37C69) (which is a Flow `SynchronizedRequestHandler`) while the Hilla one uses an endpoint. I wonder now if we can use a `SynchronizedRequestHandler` in Hilla too 🤔